api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -35,9 +35,12 @@
 	private String description;
 	
 	private Set<Integer> memberIds;
```

Comment:
@wluyima Removing `memberIds` will obviously affect the `Cohort` Constructors using this field. Should I be refactoring the Constructors to use `members` in some way?